tests/model_serving/model_server/private_endpoint/infra.py

```diff
@@ -0,0 +1,40 @@
+from typing import Generator, Dict, Optional
+
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+from ocp_resources.project_project_openshift_io import Project
+from ocp_resources.project_request import ProjectRequest
+
+TIMEOUT_2MIN = 2 * 10
+TIMEOUT_6MIN = 6 * 10
+
+
+def create_ns(
+    name: str,
+    unprivileged_client: DynamicClient = None,
+    labels: Optional[Dict[str, str]] = None,
+    admin_client: DynamicClient = None,
+    teardown: bool = True,
+    delete_timeout: int = TIMEOUT_6MIN,
+) -> Generator[Namespace] | Generator[Project]:
+    if not unprivileged_client:
```

Comment:
we currently do not have `unprivileged_client`, please add the option in `tests/conftest.py`